docs-source/content/faq/resource-settings.md

```diff
@@ -0,0 +1,177 @@
+ - [CPU resource considerations](#cpu-resource-considerations)
+ - [Operator sample heap and resource configuration](#operator-sample-heap-and-resource-configuration)
+ - [Configuring CPU affinity](#configuring-cpu-affinity)
+ - [Measuring JVM heap, pod CPU, and pod memory](#measuring-jvm-heap-pod-cpu-and-pod-memory)
+ - [References](#references)
+
+### Introduction
+
+An operator creates a pod for each WebLogic Server instance and each pod will have a container. You can tune pod container memory and/or CPU usage by configuring Kubernetes resource requests and limits, and you can tune a WebLogic JVM heap usage using the `USER_MEM_ARGS` environment variable in your domain resource. A resource request sets the minimum amount of a resource that a container requires. A resource limit is the maximum amount of resource a container is given and prevents a container from using more than its share of a resource. Additionally, resource requests and limits determine a pod's Quality of Service.
+
+This FAQ discusses tuning these parameters so WebLogic servers can run efficiently.
+
+### Setting resource requests and limits in a domain resource
+
+You can set Kubernetes memory and CPU requests and limits in a [domain resource]({{< relref "/userguide/managing-domains/domain-resource" >}}) using its `spec.serverPod.resources` stanza, and you can override the setting for individual WebLogic servers or clusters using the `serverPod.resources` element in `spec.adminServer`, `spec.clusters`, and/or `spec.managedServers`. For example: 
+
+```
+  spec:
+    serverPod:
+      requests:
+        cpu: "250m"
+        memory: "768Mi"
+      limits:
+        cpu: "2"
+        memory: "2Gi"
+```
+
+Limits and requests for CPU resources are measured in cpu units. One cpu, in Kubernetes, is equivalent to 1 vCPU/Core for cloud providers and 1 hyperthread on bare-metal Intel processors. An `m` suffix in a CPU attribute indicates 'milli-CPU', so `250m` is 25% of a CPU. 
+
+Memory can be expressed in various units, where one `Mi` is one IEC unit mega-byte (1024^2), and one `Gi` is one IEC unit giga-byte (1024^3).
+
+See also [Managing Resources for Containers](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/), [Assign Memory Resources to Containers and Pods](https://kubernetes.io/docs/tasks/configure-pod-container/assign-memory-resource) and [Assign CPU Resources to Containers and Pods](https://kubernetes.io/docs/tasks/configure-pod-container/assign-cpu-resource) in Kubernetes documentation.
+
+### Determining pod Quality Of Service
+
+A pod's Quality of Service (QoS) is based on whether it's configured with resource requests and limits:
+
+- **Best Effort QoS** (lowest priority): If you don't configure requests and limits for a pod, then the pod is given a `best-effort` QoS. In cases where a node runs out of non-shareable resources, a Kubernetes `kubelet` default out-of-resource eviction policy evicts running pods with the `best-effort` QoS first.
```

Comment:
"a Kubernetes `kubelet` default out-of-resource" -> "the default"

Reply:
Changed the wording to "the default".